src/Interfaces/AbstractPolyhedron_functions.jl

```diff
@@ -690,7 +690,7 @@ end
 
 function linear_map_inverse(Minv::AbstractMatrix{N},
                             P::AbstractPolyhedron{N}) where {N}
-    @assert size(Minv, 2) == dim(P) "a linear map of size $(size(Minv)) " *
+    @assert size(Minv, 1) == dim(P) "a linear map of size $(size(Minv)) " *
```

Comment:
It's the same.. :)  If `Minv` is invertible => `m = n`.

Reply:
It would be more sensible to check that `Minv` is square though, since `_linar_map_inverse_hrep` doesn't make that check i think.